stable/redis/README.md

```diff
@@ -147,6 +147,7 @@ The following table lists the configurable parameters of the Redis chart and the
 | `securityContext.enabled`                     | Enable security context (both redis master and slave pods)                                                                                          | `true`                                                  |
 | `securityContext.fsGroup`                     | Group ID for the container (both redis master and slave pods)                                                                                       | `1001`                                                  |
 | `securityContext.runAsUser`                   | User ID for the container (both redis master and slave pods)                                                                                        | `1001`                                                  |
+| `securityContext.sysctls`                     | Set namespaced sysctls for the container (both redis master and slave pods)                                                                         | `nil`                                                   |
```

Comment:
Should the description in the README be changed now that `securityContext.sysctls` is supported?

https://github.com/helm/charts/blob/b1ba7434ea7e1bd55145b42569ed21e12be59b8e/stable/redis/README.md#L395-L409

Reply:
I would add an additional paragraph about the new setting and keep the old one. Not all kubernetes versions support sysctls (support since 1.12, see https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster).